dev/make-root/rntuple_split_3e4.C

```diff
@@ -0,0 +1,31 @@
+R__LOAD_LIBRARY(ROOTNTuple)
```

Comment:
Hey, please prepare a header explaining what this is about - easier for anyone looking quickly - similar to what is at https://github.com/scikit-hep/scikit-hep-testdata/blob/main/dev/make-root/rntuple_int_5e4.C, for example. Advance thanks.